ci/settings/master-branch.yml

```
@@ -1,6 +1,8 @@
-project: concourse-pypi-resource-pipeline-template
+project: concourse-pypi-resource-pipeline
 docker_tag: latest
-docker_tag_add_file: version/date.txt
+docker_tag_add_file: version/version
+semver_initial_version: 0.9.0
+semver_key: artifacts/concourse-example-pypi/version
```

Shouldn't that be:
```suggestion
semver_key: final/concourse-example-pypi/version
```
as it's for the master branch it should be persisted but stuff in `artifacts` get pruned after some time.

---

Ah I should change the name as I don't use anymore the concourse-example package. Also the `version` used for building the test wheel will not be bumped, just used to have a version. So finally this version is really used and bumped for `docker_additional_tags`. In this case I could use `artifact` for the feature pipeline and `final` for master.